installer/linux/mime-type.in

```diff
@@ -7,5 +7,8 @@
         <glob-deleteall/>
         <glob pattern="*.vtf"/>
         <glob pattern="*.VTF"/>
+        <magic priority="50">
```

Comment:
This should also include console magics. `XTF\0` (not supported yet but will be soon), `VTFX`, `VTF3`. [See also](https://github.com/craftablescience/sourcepp/blob/4436a77e759f88429e6746295f9f51526ba2529a/include/vtfpp/VTF.h#L21)